docs/components/header.html

```diff
@@ -44,21 +44,36 @@
   </div>
   {{#if showHeaderBanner}}
   <div class="header-promo is-hidden-touch">
-    <div class="container">
-      <div class="">
+    <div class="container content">
+      <div class="header-promo-content">
         <h1>Join our Amazing Community</h1>
         <p class="is-intro">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt</p>
-        <div class="field is-grouped">
-          <p class="control is-expanded">
-            <input class="input" type="email">
-            <label>Enter you email</label>
-          </p>
-          <p class="control">
-            <button class="button is-default">Subscribe</button>
-          </p>
+        <div>
+          <div class="form-submit">
+            <div class="field is-grouped">
+              <p class="control is-expanded">
+                <input class="input" name="EMAIL" type="email">
+                <label>Enter you email</label>
+              </p>
+
+              <div style="position: absolute; left: -5000px;" aria-hidden="true">
```

Comment:
I suggest moving style to scss file.

Reply:
This is a special case, and should not be in styles.